src/websocket_endpoint.cpp

```diff
@@ -108,5 +138,108 @@ namespace sensr
     }     
   }
 
+  /// Verify that one of the subject alternative names matches the given hostname
+  bool WebSocketEndPoint::VerifySubjectAlternativeName(const char * hostname, X509 * cert) {
+    STACK_OF(GENERAL_NAME) * san_names = NULL;    
```

Comment:
I assume `STACK_OF(GENERAL_NAME)` is a macro that provides the typename, right? (Also let's change NULL to `nullptr`